### PR TITLE
feat(Hidden): Add Hidden component

### PR DESCRIPTION
**Pull request by kosanna**

### CODE REVIEW

**Comment by markdalgleish** [comment]
This is looking great! So excited to finally have a component like this 👏 

By the way, it looks like you've forgotten to export it from the main React bundle.

**Comment by kosanna** [comment]
@markdalgleish good point, will do

File: react/Hidden/Hidden.less [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+.print {
+  @media print {
+    display: none;
+  }
+}
+
+.screen {
+  display: none;
+
+  @media print {
```
Comment (markdalgleish) [inline react/Hidden/Hidden.less]: This should be `@media screen`, right?

Comment (kosanna) [inline react/Hidden/Hidden.less]: naw, it's exactly print. Because it's a `<Hidden screen>` use-case

Comment (markdalgleish) [inline react/Hidden/Hidden.less]: Oh right, yeah, sorry.

Could this be simplified to `@media screen { display: none; }` like the print styles?

**Pull request merged**